test/end-to-end/stock/stock.aggregate_consumption.page.js

```diff
@@ -0,0 +1,164 @@
+  };
+
+  /**
+   * @method setDetailled
+   */
+  page.setDetailled = async function setDetailled(rowIndex, columnIndex) {
+    const getCell = await GU.getCell(gridId, rowIndex, columnIndex);
+    await getCell.element(by.id('aggregated_details')).click();
+  };
+
+  page.setLotsDetailled = async function setLotsDetailled(lotsArray) {
+    const dateCell0 = await GU.getCell(modalGridId, 0, 1);
+    const quantityConsumptionCell0 = await GU.getCell(modalGridId, 0, 2);
+    const quantityLostCell0 = await GU.getCell(modalGridId, 0, 3);
+    await components.datePicker.set(lotsArray[0].date, dateCell0);
+    await FU.input('row.entity.quantity_consumed', lotsArray[0].quantity_consumed, quantityConsumptionCell0);
+    await FU.input('row.entity.quantity_lost', lotsArray[0].quantity_lost, quantityLostCell0);
+    await components.addItem.set(1);
+
+    const dateCell1 = await GU.getCell(modalGridId, 1, 1);
+    const quantityConsumptionCell1 = await GU.getCell(modalGridId, 1, 2);
+    const quantityLostCell1 = await GU.getCell(modalGridId, 1, 3);
+    await components.datePicker.set(lotsArray[1].date, dateCell1);
+    await FU.input('row.entity.quantity_consumed', lotsArray[1].quantity_consumed, quantityConsumptionCell1);
+    await FU.input('row.entity.quantity_lost', lotsArray[1].quantity_lost, quantityLostCell1);
+    await components.addItem.set(1);
+
+    const dateCell2 = await GU.getCell(modalGridId, 2, 1);
+    const quantityConsumptionCell2 = await GU.getCell(modalGridId, 2, 2);
+    const quantityLostCell2 = await GU.getCell(modalGridId, 2, 3);
+    await components.datePicker.set(lotsArray[2].date, dateCell2);
+    await FU.input('row.entity.quantity_consumed', lotsArray[2].quantity_consumed, quantityConsumptionCell2);
+    await FU.input('row.entity.quantity_lost', lotsArray[2].quantity_lost, quantityLostCell2);
+    await components.addItem.set(1);
+
+    const dateCell3 = await GU.getCell(modalGridId, 3, 1);
+    const quantityConsumptionCell3 = await GU.getCell(modalGridId, 3, 2);
+    const quantityLostCell3 = await GU.getCell(modalGridId, 3, 3);
+    await components.datePicker.set(lotsArray[3].date, dateCell3);
+    await FU.input('row.entity.quantity_consumed', lotsArray[3].quantity_consumed, quantityConsumptionCell3);
+    await FU.input('row.entity.quantity_lost', lotsArray[3].quantity_lost, quantityLostCell3);
+    await components.addItem.set(1);
+
+    const dateCell4 = await GU.getCell(modalGridId, 4, 1);
+    const quantityConsumptionCell4 = await GU.getCell(modalGridId, 4, 2);
+    const quantityLostCell4 = await GU.getCell(modalGridId, 4, 3);
+    await components.datePicker.set(lotsArray[4].date, dateCell4);
+    await FU.input('row.entity.quantity_consumed', lotsArray[4].quantity_consumed, quantityConsumptionCell4);
+    await FU.input('row.entity.quantity_lost', lotsArray[4].quantity_lost, quantityLostCell4);
+
+    FU.modal.submit();
+  };
+
+  page.setLots2Detailled = async function setLots2Detailled(lotsArray) {
```

Comment:
What is `setLots2`?  What is the actual action that is being performed here?

Reply:
![image](https://user-images.githubusercontent.com/8035588/109496996-b296b580-7a91-11eb-8357-38232990fa8f.png)
It's just to fill the modal of consumption detailed by date during E2E test